cli/cluster/instances.go

```diff
@@ -30,12 +30,12 @@ func ConnectToSomeRunningInstance(ctx *context.Ctx) (*connector.Conn, error) {
 		return nil, fmt.Errorf("Failed to get instances configuration: %s", err)
 	}
 
-	runningInstancesNames := getRunningInstances(instancesConf, ctx)
-	if len(runningInstancesNames) == 0 {
+	runningInstances := getRunningInstances(instancesConf, ctx)
+	instanceName, ok := GetFirstInstanceName(runningInstances)
+	if !ok {
```

Comment:
Perhaps we dont need to return 2 values from GetFirstInstanceName,
 why couldnt we just check: `if instanceName == ""{
...
}`

Reply:
Agree.